projects/fusion-ui/components/table/common/entities/table-options.ts

```diff
@@ -68,8 +68,11 @@ export interface TableRowOptions {
 
 export interface TableRowMetaData {
     readonly?: boolean;
+    inRequest?: boolean;
     disabled?: boolean;
     cellToolTip?: {[columnKey: string]: string};
+    rowspanColumnsData?: {[key: string]: number};
```

Comment:
As we don't make any data modifications, we store rowspan info here. KEY - column key, NUMBER - length values in the data array (how many rows we need to span for one input row).